src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -217,22 +217,28 @@ def post(self, request):
         try:
             identity = Identity.objects.get(
                 external_id=user_id,
-                idp__organization_id=group.organization.id,
+                idp__organization_id=0,
```

Comment:
You could also just write this using a `Q` object

```
Q(idp__organization_id=0) | Q(idp__organization_id=group.organization.id)
```

Reply:
Or `idp__organization_id__in=[0, group.organization_id]` :)

Reply:
even better!

Reply:
does the `in` respect the order?

Reply:
No, but neither does the `Q`. You'd have to add it to an `.order_by('organization_id')` or just order it in python.

Reply:
cool. this is temporary so i'll leave it as is for now